CoreScriptsRoot/Modules/Settings/Pages/GameSettings.lua

```diff
@@ -514,6 +518,7 @@ local function Initialize()
 	local function createMouseOptions()
 		local MouseSteps = 10
 		local MinMouseSensitivity = 0.2
+		local AdvancedEnabled = true -- settings():GetFFlag('AdvancedMouseSensitivityEnabled')
```

Comment:
also, we should wrap the GetFFlag in a pcall and check both the return value and if the call was successful.  Check out some of the other files in github to see how we normally get a fast flag